src/node.cc

```diff
@@ -5266,16 +5266,11 @@ v8::Local<v8::Value> Evaluate(const std::string& java_script_code) {
   MaybeLocal<v8::Script> script = v8::Script::Compile(env->context(), v8::String::NewFromUtf8(isolate, java_script_code.c_str())/*, origin*/);
   if (script.IsEmpty()) {
     ReportException(env, try_catch);
-    exit(3); //TODO jh: don't exit process when function breaks. Handle error differently.
+    return MaybeLocal<v8::Value>();
   }
 
-  Local<Value> result = script.ToLocalChecked()->Run();
-  if (result.IsEmpty()) {
-    ReportException(env, try_catch);
-    exit(4); //TODO jh: don't exit process when function breaks. Handle error differently.
-  }
-
-  return scope.Escape(result);
```

Comment:
What did this do?

Reply:
`scope.Escape(...)` pushes the given handle into the previously active scope.
This is used in order to save the result, once the script returned (since the scripts scope should be closed after it was executed).
It seems that this is needed for our usecase as well, therefore i will put it back in the code within my next commit.

Reply:
Done (see https://github.com/hpicgs/node/pull/40/commits/b0237880a6170f6db659230f5b19e5de14cec449)